### ISSUE TITLE
Ason class/AsonSerializer: Allow serializing superclass fields as well

### ISSUE THREAD

**Issue by Maxr1998** [afollestad/ason#15]
Considering I have the following class structure:

```java
public class Request {

    public final int id = 1;
    public final String type;

    public Request() {
        type = "important_type";
    }

    public static class SubRequest extends Request {

        public final String data;

        public SubRequest() {
            data = "example_data";
        }
    }
}
```

When I now run `Ason.serialize(new SubRequest()).toString(4);`, the String doesn't include the attributes of the superclass, and I can't pass in the fields manually since the `AsonSerializer` class is private and additionally the `AsonSerializer.serializeField()` package-private.

I suggest adding a method `serialize(Object object, boolean recurse)` which also includes the superclasses.

**Comment by afollestad** [comment]
Available in release 1.4.14

**Comment by Maxr1998** [comment]
Wow, thanks a lot!
